packages/cubejs-backend-shared/src/promises.ts

```diff
@@ -265,7 +265,7 @@ export const retryWithTimeout = <T>(
   );
 
 /**
- * High order function that makes to debounce multi async calls to single call at one time
+ * Debunks multiple asynchronous calls at once.
```

Comment:
I believe you mean `debounce` and not `debunks` :) 

```suggestion
 * Creates a debounced version of an asynchronous function.
```